suites/rbd/valgrind/validator/memcheck.yaml

```diff
@@ -1,4 +1,8 @@
 overrides:
+  install:
+    ceph:
+      flavor: notcmalloc
```

Comment:
Yeah.  I'm surprised this test was working without it.. maybe it's being run on client code only?  In any case, doesn't hurt.